content/en/docs/userguide/metric.md

```diff
@@ -0,0 +1,135 @@
+---
+draft: false
+linktitle: Querying Metrics from Prometheus
+menu:
+  docs:
+    parent: user guide
+    weight: 4
+title: Querying Metrics from Prometheus
+toc: true
+type: docs
+
+---
+
+### Preparation
+
+1. Install Kmesh:
+
+Please refer [quickstart](https://kmesh.net/en/docs/setup/quickstart/)
+
+Note that the following should be added to the Kmesh annotation:
+
+```console
+annotations:
+    prometheus.io/path: "status/metric"
+    prometheus.io/port: "15020"
+    prometheus.io/scrape: "true"
```

Comment:
Haven't these annotations been added to the Kmesh DaemonSet via https://github.com/kmesh-net/kmesh/pull/589 ?

Reply:
Emphasise that these annotations